src/dotnet-benchmark-client/project.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": "1.0.0-*",
+  "description": "REST APIs to run ASP.NET benchmark client.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/AspNet/Benchmarks"
+  },
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "emitEntryPoint": true,
+    "preserveCompilationContext": true
+  },
+  "dependencies": {
+    "Microsoft.AspNetCore.Mvc": "1.0.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
+    "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
+    "System.Diagnostics.Process": "4.1.0-*",
+    "Newtonsoft.Json": "8.0.*",
```

Comment:
If you're referencing MVC you shouldn't really need to reference this unless you explicitly want to float on the patch version